Dockerfile

```diff
@@ -1,11 +1,20 @@
 # don't change this line
 FROM korney4eg/nginx-php:latest
 
-# your code goes here
-# Make nginx use app.conf configuration to properly render php files
-# Make sure that index.php file is available in browser
-# expose needed ports
+RUN apt update && apt -y upgrade
```

Comment:
what  is the reason to have it here?

Reply:
I thought about keeping all packages up to date for security reasons.

Reply:
Make sence